server/games/game_results.py

```diff
@@ -75,21 +75,38 @@ def outcome(self, army: int) -> GameOutcome:
         if army not in self:
             return GameOutcome.UNKNOWN
 
-        outcomes = set(
-            r.outcome for r in self[army]
-            if r.outcome is not GameOutcome.UNKNOWN
+        voters = defaultdict(set)
+        for report in filter(
+            lambda r: r.outcome is not GameOutcome.UNKNOWN, self[army]
+        ):
+            voters[report.outcome].add(report.reporter)
+
+        if len(voters) == 0:
+            return GameOutcome.UNKNOWN
+
+        if len(voters) == 1:
+            unique_outcome = next(iter(voters.keys()))
+            return unique_outcome
+
+        sorted_outcomes = sorted(
+            voters.keys(),
+            reverse=True,
+            key=lambda x: (len(voters[x]), x.value)
```

Comment:
`x` in math is unknown to compute, the result, that we want to get. What `x` mean here? What's inside the `x`?